src/main/resources/resource/PythonProxy/PythonProxy.py

```diff
@@ -1,2 +0,0 @@
-# TODO: Implement this script fpr
-pythonproxy = runtime.start("pythonproxy","PythonProxy")...
```

Comment:
Didn't know that was still in there, wonder if there's still code elsewhere for that....

Reply:
its gone ...  this just a little cleanup